common/util.go

```diff
@@ -432,14 +436,20 @@ func readJSONInputs(c *cli.Context) ([][]byte, error) {
 
 		return inputsRaw, nil
 	} else if c.IsSet(FlagInputFile) {
-		inputFile := c.String(FlagInputFile)
-		// This method is purely used to parse input from the CLI. The input comes from a trusted user
-		// #nosec
-		data, err := os.ReadFile(inputFile)
-		if err != nil {
-			return nil, fmt.Errorf("unable to read input file: %w", err)
+		inputFiles := c.StringSlice(FlagInputFile)
+
+		args := [][]byte{}
```

Comment:
Go tip/suggestion, size your slices if you know how big they'll need to be

```suggestion
		args := make([][]byte, 0, len(inputFiles))
```

Or if you want to use the index to populate it

```suggestion
		args := make([][]byte, len(inputFiles))
```